src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write your code here
+const elements = [...document.querySelectorAll('span.population')];
+
+const populationArr = elements.map((element) => {
+  return Number(element.innerText.split(',').join(''));
+});
+
+const totalPopulation = populationArr.reduce((acc, oneCountryPopulation) => {
+  return acc + oneCountryPopulation;
+}, 0);
+
+const averagePopulation = totalPopulation / populationArr.length;
+
+document.querySelector('span.total-population').innerText =
+  totalPopulation.toLocaleString('en-US');
```

Comment:
You are using the `toLocaleString` method to format the total population, which is a good practice. However, you are hardcoding the 'en-US' locale. It would be better to use the user's locale, which can be obtained with `navigator.language`.